lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
Why are we adding description here? We should avoid using `def message` as much as we can.